src/server/game/Spells/Auras/SpellAuraEffects.cpp

```diff
@@ -6329,8 +6336,6 @@ void AuraEffect::HandlePeriodicDamageAurasTick(Unit* target, Unit* caster) const
             // 5..8 ticks have normal tick damage
         }
     }
-    else // xinef: ceil obtained value, it may happen that 10 ticks for 10% damage may not kill owner
```

Comment:
I know I'm replacing an `else` with a conditional, but the only two spell aura types that will hit this function anyways are `SPELL_AURA_PERIODIC_DAMAGE_PERCENT` and `SPELL_AURA_PERIODIC_DAMAGE`.

https://github.com/azerothcore/azerothcore-wotlk/blob/ce9a9d9345ad79dfd262dd1ebe2c98956deaa66c/src/server/game/Spells/Auras/SpellAuraEffects.cpp#L1131-L1134